site/docs/connectors/integration/event-schema-integration-connector.md

```diff
@@ -0,0 +1,104 @@
+
+This is its connection definition to use on the [administration commands that configure the Topic Integrator OMIS](/guides/admin/servers/configuring-an-integration-daemon/#configure-the-integration-services).
+
+!!! example "Connection configuration"
+```json linenums="1" hl_lines="14"
+    {
+    "class": "IntegrationServiceRequestBody",
+    "omagserverPlatformRootURL": "{{EGERIA_MDS_URL}}",
+    "omagserverName": "{{EGERIA_MDS_SERVER}}",
+    "connectorUserId": "{{EGERIA_USER}}",
+    "integrationConnectorConfigs": [
+        {
+            "class": "IntegrationConnectorConfig",
+            "connectorName": "EventSchemaIntegrator",
+            "typeName": "Event Schema Integration Connector",
+            "connection": {
+                "class": "Connection",
+                "connectorType": {
+                    "class": "ConnectorType",
+                    "connectorProviderClassName": "org.odpi.openmetadata.adapters.connectors.integration.eventschema.EventSchemaIntegrationProvider"
+                },
+                "configurationProperties": {
+                    "topicNamespace": "{{TOPIC_NAMESPACE}}"
+                },
+                "endpoint": {
+                    "class": "Endpoint",
+                    "address": "{{SCHEMA_REGISTRY_ENDPOINT}}"
+                }
+            },
+            "metadataSourceQualifiedName": "{{EVENT_SCHEMA_SOURCE_NAME}}",
+            "refreshTimeInterval": "{{EVENT_SCHEMA_CONNECTOR_REFRESH_INTERVAL}}",
+            "permittedSynchronization": "FROM_THIRD_PARTY"
+        }
+    ]
+}
+```
+Set the following variables with valid value:
+
+| Variable | Description | Sample |
+|----------|-------------|--------|
+| EVENT_SCHEMA_CONNECTOR_REFRESH_INTERVAL| Time in seconds between the refresh of the metadata by the connector| `5` |
+| EVENT_SCHEMA_SOURCE_NAME| Metadata source qualified name used for the created metadata. This must match the metadata source qualified name of the KafkaTopics to be matched against.| `my-event-metadata-source` |
+| EGERIA_MDS_URL| URL of the metadata server to use for the integration of the event schema metadata | `https://localhost:8443` |
+| EGERIA_MDS_SERVER| Name of the metadata server | `mds1` |
+| EGERIA_USER| Name of the user | `garygeeke` |
+| EGERIA_SERVER| Name of the integration server | `evtschema1` |
+| EGERIA_URL| URL of the integration server | `https://localhost:9443` |
+| TOPIC_NAMESPACE| Namespace in which the topics are found which are used to map the EventTypes to. This is used as a prefix to find the correct topic. | `customer_topics` |
+| SCHEMA_REGISTRY_ENDPOINT| The URL of the Schema Registry. | `https://my-schemaregistry:8081`|
+
+
+
+
+## Running locally for testing
+
+You will need to have an OMAG platform with a configured metadata server and activated DataManager OMAS. You will also have a Docker client installed locally and a running Docker daemon.
```

Comment:
we call the metadata server the Metadata Access Store now. Also the Metadata Access Point could be being used to talk to a remote Metadata Access Store.